lib/components/map/distance-measure.js

```diff
@@ -1,21 +0,0 @@
-import { MapControl, withLeaflet } from 'react-leaflet'
-import L from 'leaflet'
-import 'leaflet.polylinemeasure/Leaflet.PolylineMeasure.js'
-
-class DistanceMeasure extends MapControl {
-  createLeafletElement (props) {
-    return L.control.polylineMeasure({
-      unit: 'landmiles',
-      measureControlLabel: '&#x1f4cf;',
-      backgroundColor: '#f3dd2d',
-      clearMeasurementsOnStop: true
-    })
-  }
-
-  componentDidMount () {
-    const { map } = this.props.leaflet
-    this.leafletElement.addTo(map)
-  }
-}
-
-export default withLeaflet(DistanceMeasure)
```

Comment:
Does TriMet no longer need this?

Reply:
It wasn't imported anywhere, so I guess they don't use it anymore.